code/controllers/subsystem/disease.dm

```diff
@@ -12,6 +12,13 @@ SUBSYSTEM_DEF(disease)
 	if(!diseases)
 		diseases = subtypesof(/datum/disease)
 
+/datum/controller/subsystem/disease/Initialize(timeofday)
```

Comment:
This should probably use PreInit just in case

Reply:
In case of what ? Someone using pandemic pre subsystem inits ? That seems unlikely